tests/integration/test_vulnerability_detector/test_scan_results/data/vulnerabilities_different_cves.json

```diff
@@ -0,0 +1,153 @@
+
```

Comment:
Why did you create a new file instead of extending `vulnerabilities.json`

Reply:
because the structure of these files is different. In this one each CVE exists twice, one for the provider and another for NVD but with different fix versions.